frontend/tests/test-data/all_sample_files/sample.html

```
@@ -0,0 +1 @@
+Sample content
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)